File: sentimental-cash/cash.py
from cs50 import get_float

while True:
    change = get_float("Change: ")
    if change > 0:
        break

twofive = 0
ten = 0
five = 0
one = 0
change = round(change * 100)

while change > 0:
    if change >= 25:
        change -= 25
        twofive += 1
    elif change >= 10:
        change -= 10
        ten += 1
    elif change >= 5:
        change -= 5
        five += 1
    else:
        change -= 1
        one += 1

result = twofive + ten + five + one
print(result)

File: cash/cash.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int c_o;
    do
    {
        c_o = get_int("Change Owed: ");
    }
    while (c_o < 1);
    int twofive = 0;
    int ten = 0;
    int five = 0;
    int one = 0;
    while (c_o > 0)
        if (c_o >= 25)
        {
            c_o -= 25;
            twofive++;
        }

        else if (c_o >= 10)
        {
            c_o -= 10;
            ten++;
        }

        else if (c_o >= 5)
        {
            c_o -= 5;
            five++;
        }

        else
        {
            c_o -= 1;
            one++;
        }
    printf("%i\n", twofive + ten + five + one);
}
